app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False
+                 ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def death(self) -> None:
+        if self in Animal.alive:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
```

Comment:
This is a code style issue. You are using concatenation in the __repr__ method. Please use interpolation instead. In Python, it is recommended to use string formatting (interpolation) over concatenation because it is more readable and efficient. Here is an example of how you can do it: `return f'{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}'`.